JavaComparator/Comparator.java

```diff
@@ -0,0 +1,133 @@
+import java.util.ArrayList;
+
+public class Comparator {
+    String file_orig, file_tripped;
+    String file_extension;
+
+    Comparator(String Path1, String Path2) {
+        file_orig = Path1;
+        file_tripped = Path2;
+
+        if (Path1.endsWith("docx") && Path2.endsWith("docx")) {
```

Comment:
Change Path1, Path2 etc, variable names to reflect what they are referring to - originalFilePath etc.